src/style.css

```diff
@@ -0,0 +1,99 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: AvenirBold;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+}
+
+body {
+  margin-top: 0;
+}
+
+.big-bar {
+  margin: 20px 0;
+  position: relative;
+}
+
+.searchImg {
+  position: absolute;
+  width: 19px;
+  bottom: 26px;
+  left: 26px;
+}
+
+.bigSearchField {
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  color: #3d4e61;
+  background: #fff;
+  line-height: 22px;
+  height: 70px;
+  width: 100%;
+  padding-left: 62px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61,78,97,0.1);
+}
+
+.bigSearchField::placeholder {
+  font-family: Avenir, sans-serif;
+}
+
+.bigSearchField :hover {
+  box-shadow: 0 3px 8px rgba(61,78,97,0.2);
+}
+
+.bigSearchField :focus {
+  outline: none;
+  font-family: AvenirBold, sans-serif;
+  background: linear-gradient(180deg,#fff,#f6f6f7);
+  border-radius: 3px;
+}
+
+.little-bar {
+  position: relative;
+}
+
+.searchImgSmall {
+  width: 11px;
+  bottom: 16px;
+  left: 13px;
+}
+
+.smallSearchField {
+  box-sizing: border-box;
+  height: 42px;
+  width: 100%;
+  font-size: 14px;
+  font-family: Avenir, sans-serif;
+  color: #3d4e61;
+  font-weight: 300;
+  line-height: 19px;
+  padding-left: 33px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61,78,97,0.1);
+}
+
+.smallSearchField :hover {
+  box-shadow: 0 3px 8px rgba(61,78,97,0.2);
+}
+
+.smallSearchField::placeholder {
+  font-family: Avenir, sans-serif;
+}
+
+.smallSearchField :focus {
+  outline: none;
+  font-family: AvenirBold, sans-serif;
+  background: linear-gradient(180deg,#fff,#f6f6f7);
+  border-radius: 3px;
+}
```

Comment:
You duplicate a bunch of styles. [Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) the 2nd item